Chart.yaml: directory name (puppetserver-helm-chart) and chart name (puppetserver) must be the same

https://github.com/puppetlabs/puppetserver-helm-chart/actions/runs/3357111678/jobs/5562701873#step:5:11

```

0s
Run helm lint .
  helm lint .
  shell: /usr/bin/bash -e {0}
  env:
    GITHUB_TOKEN: ***
    HELM_REPO_URL: https://puppetlabs.github.io/puppetserver-helm-chart
    GITHUB_REPO: puppetlabs/puppetserver-helm-chart
Error: 1 chart(s) linted, 1 chart(s) failed
	%!s(<nil>)
Error: ] Chart.yaml: directory name (puppetserver-helm-chart) and chart name (puppetserver) must be the same

Error: Process completed with exit code 1.
```

--- a/.github/workflows/release.yaml
+++ b/.github/workflows/release.yaml
@@ -20,10 +20,10 @@ jobs:
       GITHUB_REPO: ${{ github.event.inputs.githubRepo }}
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Fetch history
         run: git fetch --prune --unshallow
-      - uses: azure/setup-helm@v1
+      - uses: azure/setup-helm@v3.4
       - name: lint
         run: |
           helm lint .

